### PR TITLE
#142: Add broadcast channel

### PR DESCRIPTION
**Pull request by euphi**
As proposed in #142 , I added a broadcast channel.

Sample implementation is at https://github.com/euphi/ESP_Heizungscontroller . (It does not make real use of it, it just logs the received message to Serial and back to MQTT).

### CODE REVIEW

File: src/Homie/Datatypes/Interface.hpp [hunk 1]
```diff
@@ -37,6 +37,7 @@ struct Interface {
   } reset;
 
   GlobalInputHandler globalInputHandler;
+  BroadcastHandler broadcastHandler;
```
Comment (marvinroger) [inline src/Homie/Datatypes/Interface.hpp]: `BroadcastHandler` is defined nowhere

File: src/Homie.cpp [hunk 2]
```diff
@@ -238,6 +239,14 @@ HomieClass& HomieClass::setGlobalInputHandler(GlobalInputHandler inputHandler) {
   return *this;
 }
 
+HomieClass& HomieClass::setBroadcastHandler(BroadcastHandler broadcastHandler) {
+  _checkBeforeSetup(F("setGlobalInputHandler"));
```
Comment (marvinroger) [inline src/Homie.cpp]: Can you replace this with `setBroadcastHandler`?

File: src/Homie/Datatypes/Interface.hpp (hunk 1 above)
Comment (euphi) [inline src/Homie/Datatypes/Interface.hpp]: Seems I missed to add some changes for commit. 

I'm on vacation again, so I'll fix after October 20th.

File: src/Homie.cpp (hunk 2 above)
Comment (euphi) [inline src/Homie.cpp]: Oh, of course.

**Comment by euphi** [comment]
When I'm back, I'll merge your changes and create a new pull request

**Comment by marvinroger** [comment]
Nevermind, I just fixed your PR. Thanks! 👍

**Pull request merged**